setup.py

```diff
@@ -32,7 +32,8 @@
     install_requires=[
         "bravado-core >= 4.2.2",
         "crochet >= 1.4.0",
-        "fido >= 2.1.0, < 3.2.0",
+        "fido >= 2.1.0, <= 3.2.0",
```

Comment:
I'm expecting another change of contract when implementing a timeout mechanism (planning on providing a timeout at client creation to be used as a default for http calls).
But I see your point. This restriction is not necessary but just a safety net, removing.